governance-proposal/TSC-Charter-Draft.md

```diff
@@ -0,0 +1,103 @@
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
+* Development process and any coding standards
+* Mediating technical conflicts between Collaborators or Foundation projects
+
+The TSC will define Node.js Foundation’s release vehicles and serve as Node.js Foundation’s primary technical liaison body with external open source projects, consortiums and groups.
+
+## Section 6. Node.js Foundation Operations.
+
+The TSC will establish and maintain a development process for Node.js Foundation Projects. The development process will establish guidelines for how the developers and community will operate. It will, for example, establish appropriate timelines for TSC review (e.g. agenda items must be published at least a certain number of hours in advance of a TSC meeting).
+
+There will be multiple Projects under the Node.js Foundation organized by modules or subsystems. The TSC is responsible for organizing the Project structure, including possibly the creation and alignment of sub-Projects. Each Project must be within such policies as may be set by the Board, have a well-defined scope and must work within that scope. The development process will provide for Projects to follow the lifecycle process as described in the Project Lifecycle document. The development process will include a process for the TSC to oversee and approve changes in the lifecycle of a Project, which will include consideration of the following criteria:
+
+* Cleanliness of code base
+* Ample and diverse Contributors and Collaborators to assure vitality of the project.
+* Stability (e.g. presence of test suites, stable APIs and use of an appropriate source-code control system).
+* Predictability of releases
+* Alignment with Node.js Foundation’s goals and priorities.
+
+The TSC and entire technical community will follow any processes as may be specified by the Board relating to the intake and license compliance review of contributions, including the Node.js Foundation IP Policy.
+
+## Section 7. Elections
+
+Leadership roles in Node.js Foundation will be peer elected representatives of the community.
+
+For election of persons (TSC Chairperson, Maintainers, etc.) a multiple-candidate method should be used, e.g.:
+
+* [Condorcet](http://en.wikipedia.org/wiki/Condorcet_method) or
+* [Single Transferable Vote](http://en.wikipedia.org/wiki/Single_transferable_vote)
+
+Multiple-candidate methods may be reduced to simple election by plurality when there are only two candidates for one position to be filled. No election is required if there is only one candidate and no objections to the candidates election. Elections shall be done within the Projects by the Collaborators active in the Project.
+
+Each Core Project’s Collaborators shall elect one Maintainer from the Collaborators on the project to serve on the TSC. There may be only one Maintainer per Core Project that shall be nominated and elected by the Collaborators within the Core Project.
```

Comment:
It would be up to the TSC to evaluate in the current model. It sets the goal/expectation, but leaves the TSC empowered to define the structure. 6-12 was chosen as a range of what would ensure diversity but also make it manageable to easily communicate and operate as a cohesive working team.